packages/codegen/package.json

```diff
@@ -31,6 +31,7 @@
   },
   "homepage": "https://github.com/xataio/client-ts/blob/main/codegen/README.md",
   "dependencies": {
+    "@xata.io/client": "workspace:*",
```

Comment:
how come this changed?

Reply:
We didn't use anything from the client before and we are now using it.